bql/semantic/expression_test.go

```diff
@@ -387,6 +387,27 @@ func TestNewEvaluator(t *testing.T) {
 			err:  false,
 			want: false,
 		},
+		{
+			id: "?foo = \"text\"^^type:text",
+			in: []ConsumedElement{
```

Comment:
Could you also add a few test on operation correctness in https://github.com/google/badwolf/blob/master/bql/planner/planner_test.go

Reply:
Done